theme_avantgarde/views/customizations.xml

```diff
@@ -28,7 +28,7 @@
 </template>
 
 <!-- ======== BANNER ======== -->
- <template id="s_cover" inherit_id="website.s_cover" name="Avantgarde s_cover">
```

Comment:
Meh... that fail :|

Did you speak with the designers to know it was supposed to be banner here? What actually happened with two s_cover defined? The first one was simply ignored? If yes, should not it be simply removed since nobody actually saw there was a mistake? :grimacing:

Reply:
Hello @smottet , it was supposed to be `s_banner`, thanks.
Sorry for the headache.